ops/terraform/locals.tf

```diff
@@ -1,25 +1,29 @@
 locals {
-  environment = terraform.workspace
+  environment    = terraform.workspace
+  ocr-api        = "ocr"
+  middleware-api = "middleware"
   init = {
     environment = local.environment
     location    = "eastus2"
   }
   dev = {
     dev = {
-      vnetcidr      = "10.0.0.0/16"
-      appsubnetcidr = "10.0.1.0/24"
-      websubnetcidr = "10.0.2.0/24"
-      lbsubnetcidr  = "10.0.3.0/24"
-      dbsubnetcidr  = "10.0.4.0/24"
+      vnetcidr             = "10.0.0.0/16"
+      ocrsubnetcidr        = "10.0.1.0/24"
+      middlewaresubnetcidr = "10.0.2.0/24"
```

Comment:
Looks like you changed `appsubnetcidr` to `ocrsubnetcidr`, added `middlewaresubnetcidr`, and changed the order and number scheme of the subnets.  Not sure why it all shows green but wanted to make sure this is the only changes.  Or am I missing something?

Reply:
it all shows green because github's dif checking is very confusing. It did that because each variables cidr changed. I could have added the middleware cidr at the end but wanted it to be next to the ocr's cidr.